### DISCUSSION TITLE
Allure-cucumberjs - How to "reactivate" the console standard output ?

### DISCUSSION THREAD

**Discussion by harmin-parra** [allure-framework/allure2#2739] [Questions & Support] [answered]
When I execute **Node.js** cucumber tests without the allure listener, I get an output like this after executing the tests:
```
.........

3 scenarios (3 passed)
9 steps (9 passed)
0m05.329s (executing steps: 0m05.285s)
```
When I add the **Allure** listener, I don't get output in the console when executing the tests

How to re-activate the standard output when using the **Allure** listener ?

**Comment by baev** [comment, marked as answer]
CucumberJS has super weird formatter restrictions:

> Some notes on specifying Formatters:
> 
> If multiple formatters are specified with the same output, only the last is used.
> If no formatter for stdout is specified, we default to the progress formatted.

By default, when you specify the formatter (either via cli args or config) you need to specify the output file:

```bash
$ cucumber-js --format format-name:path/to/output/file.txt
```


All the formatters without the output file specified are considered to have stdout as the output. Only the latest specified formatted per output will be used; the other will be ignored. If no formatter uses `stdout` output, cucumber-js will use the `progress` formatter (which prints the output you're missing).

The workaround is to specify just any non-existing path as the output for the allure formatter:

```bash
$ cucumber-js --format allure-cucumberjs/reporter:ignored.txt
```

Or via config:

```json
{
  "default": {
    "parallel": 2,
    "format": [["allure-cucumberjs/reporter", "ignored.txt"]],
  }
}
```

This will create an empty file named `ignored.txt` thought, which I guess you should ignore ;)
